src/chainbridgeConfig.ts

```diff
@@ -50,78 +50,35 @@ export type ChainbridgeConfig = {
 export const chainbridgeConfig: ChainbridgeConfig = {
   // Local GETH <> Local Substrate
   chains: [
-    // {
-    //   chainId: 0,
-    //   networkId: 5,
-    //   name: "Ethereum - Local",
-    //   decimals: 18,
-    //   bridgeAddress: "0x62877dDCd49aD22f5eDfc6ac108e9a4b5D2bD88B",
-    //   erc20HandlerAddress: "0x3167776db165D8eA0f51790CA2bbf44Db5105ADF",
-    //   rpcUrl: "http://localhost:8545",
-    //   type: "Ethereum",
-    //   nativeTokenSymbol: "ETH",
-    //   tokens: [
-    //     {
-    //       address: "0x21605f71845f372A9ed84253d2D024B7B10999f4",
-    //       name: "TOKEN",
-    //       symbol: "TOKEN",
-    //       imageUri: ETHIcon,
-    //       resourceId:
-    //         "0x000000000000000000000000000000c76ebe4a02bbc34786d860b355f5a5ce00",
-    //     },
-    //   ],
-    // },
-    // {
-    //   chainId: 1,
-    //   networkId: 2,
-    //   name: "Substrate - Local",
-    //   decimals: 18,
-    //   rpcUrl: "ws://localhost:9944",
-    //   type: "Substrate",
-    //   nativeTokenSymbol: "DOT",
-    //   chainbridgePalletName: "chainBridge",
-    //   bridgeFeeFunctionName: "tokenTransferFee",
-    //   transferPalletName: "example",
-    //   transferFunctionName: "transferNative",
-    //   typesFileName: "bridgeTypes.json",
-    //   tokens: [
-    //     {
-    //       address: "substrate-native",
-    //       name: "DOT",
-    //       symbol: "DOT",
-    //       resourceId: "substrate-native",
-    //     },
-    //   ],
-    // },
     {
-      chainId: 0,
-      networkId: 42,
+      chainId: 5,
+      networkId: 5,
       name: "Ethereum",
       decimals: 18,
-      bridgeAddress: "0x478ab279Ac5F4bd69382D34cF2382606E6208eFc",
-      erc20HandlerAddress: "0x3483c3a1Af5e78AE5AaB07de3Ea57b6F3877745F",
-      rpcUrl: "wss://kovan.infura.io/ws/v3/e199aa0da7e54bd9be94de96ea753127",
+      bridgeAddress: "0xa806cA3bD88F790744462cBC34c40EDd5b8dc2Dd",
+      erc20HandlerAddress: "0xf934Bfc8B5241b6C9e0DfC9A329AD687e79c5498",
+      rpcUrl: "https://goerli.infura.io/v3/2532f2073c2d4a34b5b9544c76a3de8b",
```

Comment:
Which Infura is using here? We should create a separate project for it

Reply:
Moved config changes to seperate PR